usaspending_api/download/v2/download_column_historical_lookups.py

```diff
@@ -1567,11 +1561,8 @@
             [
                 ("owning_agency_name", "treasury_account__funding_toptier_agency__name"),
                 ("reporting_agency_name", "submission__reporting_agency_name"),
-                (
-                    "last_reported_submission_period",
-                    "last_reported_submission_period",
-                ),  # Column is annotated in account_download.py
-                ("allocation_transfer_agency_identifier_code", "treasury_account__allocation_transfer_agency_id"),
+                ("submission_period", "submission_period",),  # Column is annotated in account_download.py
+                ("allocation_transfer_agency_identifer_code", "treasury_account__allocation_transfer_agency_id"),
```

Comment:
Was this intentional?
```suggestion
                ("allocation_transfer_agency_identifier_code", "treasury_account__allocation_transfer_agency_id"),
```

Reply:
OMG PLEASE DON'T LET THIS SPELLING MISTAKE BACK INTO THE CODE BASE!